FLEDGE.md

```diff
@@ -674,6 +701,7 @@ The arguments to this function are:
       'componentSeller': 'https://www.some-other-ssp.com',
       'interestGroupOwner': 'https://www.example-dsp.com/',
       'renderUrl': 'https://cdn.com/url-of-winning-creative.wbn',
+      'renderSize': {width: 100, height: 200},
```

Comment:
Mind adding a little bullet point above on line ~694 mentioning that this comes from the auction config if specified there?

Reply:
This comes from `generateBid` (maybe `scoreAd` is after that?) in the exact same way as `renderUrl`, which doesn't seem to have a bullet point around there.

It actually looks like `renderSize` might be missing from `scoreAd`'s browser signals...

Reply:
Added the fields to scoreAd, and added a comment about `/* if specified in the bid */` here. Let me know if that's enough.